pkgs/by-name/li/libjaylink/package.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall = ''
-    install -Dm644 contrib/99-libjaylink.rules $out/lib/udev/rules.d/libjaylink.rules
+    install -Dm644 contrib/60-libjaylink.rules $out/lib/udev/rules.d/libjaylink.rules
```

Comment:
Unrelated to this PR, but why do we remove the udev rule ordering? If it uses any TAG+="uaccess" it must be run before rule 73-seat-late.rule (from memory).

Reply:
Actually, that may be related to this PR now that upstream changed the rule order. There might be a reason to that.

Please add a commit to preserve the order?

Reply:
At some point I was told to do it that way, because NixOS wouldn't care about the ordering anyway. I can't remember the place anymore where we discussed this, but we did test that everything works.

However, this PR is just supposed to be a version bump and to enable automatic version updates. It doesn't apply any changes to the ordering. If it's an actual problem, then we should open a dedicated PR for that and discuss it there.

Reply:
I'm marking this thread as resolved since this issue is unrelated to this PR.

Reply:
Did you test that the udev rules work after this PR too? (Remove old rules and reboot.)

Reply:
I checked and this PR will cause a regression if the udev rule order is not fixed: https://repo.or.cz/libjaylink.git/commit/da0e8cd40dfdadd6226c9c9a146c8be25479cae1

Reply:
Next time, please don't dismiss reviews so quickly.